app/frontend/src/features/auth/email/ChangeEmailPage.tsx

```diff
@@ -55,7 +63,9 @@ export default function ChangeEmailPage() {
 
   return (
     <>
-      <PageTitle>Change email</PageTitle>
+      <PageTitle>{CHANGE_NAME_GENDER}</PageTitle>
```

Comment:
~`CHANGE_EMAIL`? I think the test is failing here - maybe will pass if you use "Change email"~ See below comment.